pkg/trace/agent/agent.go

```diff
@@ -272,11 +268,9 @@ func (a *Agent) runSamplers(pt ProcessedTrace) (sampled bool, rate float64) {
 
 	if traceContainsError(pt.Trace) {
 		sampledScore, rateScore = a.ErrorsScoreSampler.Add(pt)
```

Comment:
1. Let's remove these variable declarations at the top of this function and instead just do:
```suggestion
		scoreSampled, scoreRate := a.ErrorsScoreSampler.Add(pt)
```

2. I think it's more correct to turn the words around in the names - what do you think? It sounds better to say this is the "score's rate" or the "score's sampled" rather than the "sampled's score" or the "rate's score" which is incorrect. If you agree, please change the other two variables too to `prioritySampled` and `priorityRate`. 

I don't feel strongly about 2 so I'm ok with whatever you decide.